pom.xml

```diff
@@ -133,6 +134,11 @@
     	<version>1.39</version>
     	<scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jenkins-ci.plugins</groupId>
```

Comment:
won't this make the plugin require credentials plugin now?

Reply:
yes, but credentials plugin is standard, you will hardly find anyone that hasn't installed it

Reply:
you are right, I was thinking about credentials-binding (which is still more popular than I thought)